client/src/Components/Widget0.js

```diff
@@ -61,30 +66,69 @@ class Widget0 extends React.Component {
          setInterval(function(){
              fetch('/api/trackleads')
              .then(res => res.json())
-             .then(data => console.log(data))
+             .then(function(data){
+
+                 let {mapStyle} = this.state;
+
+                 var trackleaders = data.trackleaders_aggregate_feed.trackleaders_feed;
+                 var pointData ;
+                 var points = [];
+                 
+                 trackleaders.forEach(trackleader => {
+                    pointData = {
+                        type: 'Point',
+                        coordinates: [parseFloat(trackleader.message[0].longitude[0]) , parseFloat(trackleader.message[0].latitude[0])]
+                    };
+
+                    points.push(pointData)
```

Comment:
this creates an array of `point` geojson features.  You'll want to wrap that array into a `featureCollection` according to the geojson spec to be valid:

```
{
"type": "FeatureCollection",
"features": [{myFeatures]]
}
```

See https://docs.mapbox.com/mapbox-gl-js/example/multiple-geometries/ for reference on a valid geojson object.